ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml

```diff
@@ -24,6 +24,19 @@ build_root:
     name: apollo-ci
     namespace: stackrox
     tag: stackrox-test-0.3.56
+releases:
+  ec:
+    prerelease:
+      product: ocp
+      stream: 4-dev-preview
+      version_bounds:
+        lower: 4.1.0
+        upper: 4.999.0
+  latest:
+    candidate:
+      product: ocp
+      stream: ci
+      version: "4.13"
```

Comment:
This seems to be unused or does it play some role?

Reply:
This is required to get the job to run. I don't really understand the `releases` stanza and _should_ probably read the docs more.